src/scales/interpolatedColorScale.ts

```diff
@@ -56,49 +56,56 @@ export module Scales {
       ]
     };
 
+    private _colorRange: string[];
+    private _colorScale: D3.Scale.QuantitativeScale;
+
     /**
-     * Converts the string array into a d3 scale.
-     *
-     * @param {string[]} colors an array of strings representing color
-     *     values in hex ("#FFFFFF") or keywords ("white").
+     * An InterpolatedColorScale maps numbers to color strings.
+     * 
+     * @param {string | string[]} colors an array of strings representing color
+     *     values in hex ("#FFFFFF") or keywords ("white"). Default is "reds"
      * @param {string} scaleType a string representing the underlying scale
-     *     type ("linear"/"log"/"sqrt"/"pow")
+     *     type ("linear"/"log"/"sqrt"/"pow"). Defaults to "linear"
      * @returns {D3.Scale.QuantitativeScale} The converted QuantitativeScale d3 scale.
      */
-    private static _getD3InterpolatedScale(colors: string[], scaleType: string): D3.Scale.QuantitativeScale {
-      var scale: D3.Scale.QuantitativeScale;
+    constructor(colorRange: string | string[] = "reds", scaleType = "linear") {
```

Comment:
Can we make this take only a `string[]`, and then have the default arrays of colors stored as `static` fields?